src/app/product-form/product-form.component.html

```diff
@@ -0,0 +1,64 @@
+<form [formGroup]="productForm">
+  <div class="inputBorder">
+    <mat-form-field>
+      <mat-label>Name</mat-label>
+      <input matInput placeholder="Name of the product" [defaultValue]="productForm.get('name')" formControlName="name">
```

Comment:
You don't need a defaultValue here. If you initialize the form control with a value, it will be displayed by default.